int64 in
---
bool success
sensor_msgs/PointCloud2 cloud
